logback-access/src/main/java/ch/qos/logback/access/jetty/RequestLogImpl.java

```diff
@@ -104,223 +100,155 @@
  *         &lt;param name=&quot;Subject&quot; value=&quot;Last Event: %statusCode %requestURL&quot; /&gt;
  *         &lt;param name=&quot;To&quot; value=&quot;server_admin@domain.org&quot; /&gt;
  *      &lt;/appender&gt;
- *      &lt;appender-ref ref=&quot;SMTP&quot; /&gt;
+ *      &lt;appender-ref ref=&quot;SMTP&quot; /&gt; 
  *    &lt;/configuration&gt;
  * </pre>
- *
+ * 
  * @author Ceki G&uuml;lc&uuml;
  * @author S&eacute;bastien Pennec
  */
 public class RequestLogImpl extends ContextBase implements RequestLog,
-        AppenderAttachable<IAccessEvent>, FilterAttachable<IAccessEvent> {
+    AppenderAttachable<AccessEvent>, FilterAttachable<AccessEvent> {
 
   public final static String DEFAULT_CONFIG_FILE = "etc" + File.separatorChar
-          + "logback-access.xml";
+      + "logback-access.xml";
 
-  AppenderAttachableImpl<IAccessEvent> aai = new AppenderAttachableImpl<IAccessEvent>();
-  FilterAttachableImpl<IAccessEvent> fai = new FilterAttachableImpl<IAccessEvent>();
-  String fileName;
```

Comment:
It's [Codacy](https://www.codacy.com/app/tony19/logback/dashboard?bid=3222618).

fyi, the link in the "issue found" comment takes you the tool :smile: (not sure why it results in "file not found")